src/Pages/Projects/Projects.tsx

```diff
@@ -74,57 +120,78 @@ const Projects = () => {
-            <table className="table mt-3  ">
-              <thead>
-                <tr>
-                  <th className='ps-3'>Id</th>
-                  <th>Title</th>
-                  <th>Description</th>
-                  <th>CreationDate</th>
-                  <th>Actions</th>
-                </tr>
-              </thead>
-              <tbody>
-
-                {tableData?.data?.map((data: any, index: number) =>
-
-                  <tr key={data?.id} >
-                    <td data-cell="id" className='ps-3'>{index + 1}</td>
-                    <td data-cell="title">{data?.title}</td>
-                    <td data-cell="description" className='text-truncate'>{data?.description}</td>
-                    <td data-cell="creationDate ">{moment(data?.creationDate).format("Do MMM YY")}</td>
-
+        {!isLoading && tableData?.data?.length > 0 &&
+          <table className="table mt-3  ">
+            <thead>
+              <tr>
+                <th className='ps-3'>Id</th>
+                <th>Title</th>
+                <th>Description</th>
+                <th>CreationDate</th>
+                {userRole === "Manager" ? <th>Actions</th> : ""}
+              </tr>
+            </thead>
+            <tbody>
+
+              {tableData?.data?.map((data: any, index: number) =>
+
+                <tr key={data?.id} >
+                  <td data-cell="id" className='ps-3'>{index + 1}</td>
+                  <td data-cell="title">{data?.title}</td>
+                  <td data-cell="description" className='text-truncate'>{data?.description}</td>
+                  <td data-cell="creationDate ">{moment(data?.creationDate).format("Do MMM YY")}</td>
+
+
+
+                  {userRole === "Manager" ? <>
                     <td data-cell="actions" className='action align-items-center gap-3 '>
-
-                      <button className={`delete text-center text-danger pointer`}onClick={() => showModal(data.id)} >
+                      <button className={`delete text-center text-danger pointer`} onClick={() => showModal(data.id)} >
                         <Trash size={'20px'} />
                       </button>
 
                       <Link to={`/dashboard/edit-project/${data?.id}`} className="edit text-info pointer d-inline-block  ms-3 me-2 text-center">
                         <Edit size={'20px'} />
                       </Link>
 
+                    </td>
+                  </> : ""}
+
 
 
-                    </td>
-                  </tr>)}
 
-              </tbody>
+                </tr>)}
 
-            </table>
-          </div>
+            </tbody>
 
+          </table>
         }
+                <nav className='page' aria-label="Page navigation example ">
+          <ul className="pagination m-auto mb-4">
+            {Array(tableData?.totalNumberOfPages).fill(0).map((_, i) => i + 1).map((pageNo) =>
+              <li  onClick={()=>pagination(pageNo,searchParams)} key={pageNo} className='page-item'>
```

Comment:
the pagination is not reachable by keyboard users, as we have a list item that have an onClick (not a button) and the `a` element doesn't have an `href` attribute, so it will not be focusable as well.